recipes/ipychart/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "ipychart" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ac01fbca69039e5631d1f2579edca80f47f15da3132df9aa7259cafd9e7e2846
```

Comment:
according to pypi, the `sha256` should be `0e188841f87379bf01105fd6b5d4649a8183d6de2be2c8b7677ea2b7d750ce7c`